pkg/stub/handler.go

```diff
@@ -1338,9 +1315,14 @@ func (h *Handler) Handle(event v1.Event) error {
 			}
 			logrus.Debugf("Handle Reason field set to %s", progressing.Reason)
 			cfg.ConditionUpdate(progressing)
-			cfg.Spec.Version = v1.GitVersionString()
-			logrus.Debugf("SDKUPDATE progressing true update")
-			err = h.crdwrapper.Update(cfg)
+
+			// now that we employ status subresources, we can't populate
+			// the conditions on create; so we do initialize here, which is our "step 1"
+			// of the "make a change" flow in our state machine
```

Comment:
hm. I kinda thought you can `create` an object w/ status fields populated, you just can't do an `update` against status fields.

Reply:
(but if this works, it's fine)